### PR TITLE
Update readonly workload configuration

### PR DESCRIPTION
**Pull request by suraj-pasuparthy**
Updating the readonly benchmark configuration file with new QPS and dataset.

### CODE REVIEW

File: data/throughput_benchmarks/workloadc.yaml [hunk 1]
```diff
@@ -20,31 +20,31 @@ benchmarks:
...
-      ycsb_record_count: 100000000
+      # Data: 500M 1kb rows (500GB total)
+      ycsb_record_count: 500000000
       ycsb_field_count: 1
       ycsb_field_length: 1000
       
-      # Use 25 threads/VM in Load phase
-      ycsb_preload_threads: '25'
+      # Use 32 threads/VM in Load phase
+      ycsb_preload_threads: '32'
```
Comment (bfarayev) [inline data/throughput_benchmarks/workloadc.yaml:35]: I'm not against this change, just worried that it creates a diff here compared to other workload files, which can confuse the users? I know that users can set it to what they want and here we just want to provide sensible defaults.

File: data/throughput_benchmarks/workloadc.yaml [hunk 2]
```diff
@@ -20,31 +20,31 @@ benchmarks:
       cloud_spanner_config: regional-us-west1
       cloud_spanner_nodes: 3
 
-      # GCE Provisioning: 5 In-Region VMs per Spanner Node
-      ycsb_client_vms: 15
+      # GCE Provisioning: 30 In-Region VMs per Spanner Node
+      ycsb_client_vms: 30
```
Comment (bfarayev) [inline data/throughput_benchmarks/workloadc.yaml:24]: Given that Cloud Spanner node size will be 3 (L:21), should we either say 10 In-Region VMs per Spanner Node or set the client VM count to 90?

File: README.md [hunk 3]
```diff
@@ -389,7 +389,7 @@ Example: Write-only throughput benchmark command:
 
 ```sh
 $HOME/PerfKitBenchmarker/pkb.py \
---benchmark_config_file=$HOME/spanner-benchmarks-tutorial/data/data/throughput_benchmarks/workloadx.yaml \
+--benchmark_config_file=$HOME/spanner-benchmarks-tutorial/data/throughput_benchmarks/workloadx.yaml \
```
Comment (bfarayev) [inline README.md]: this could be in a separate commit :)

File: data/throughput_benchmarks/workloadc.yaml [hunk 4]
```diff
@@ -20,31 +20,31 @@ benchmarks:
...
       ycsb_timelimit: 1800
-      ycsb_operation_count: 100000000 # Opcount high so we hit timelimit
-      ycsb_threads_per_client: '25'
+      ycsb_operation_count: 2147483640 # Opcount high so we hit timelimit
+      ycsb_threads_per_client: '32'
 
-      # Target 2,200 QPS/VM (1,100 QPS / Spanner Node)
-      ycsb_run_parameters: target=2200,requestdistribution=zipfian,dataintegrity=True
+      # Target 1400 QPS/VM ( 14,000 QPS / Spanner Node)
+      ycsb_run_parameters: target=1400,requestdistribution=zipfian,dataintegrity=True
```
Comment (bfarayev) [inline data/throughput_benchmarks/workloadc.yaml:47]: Not related to the change here but I think the previous number here was wrong. 2,200 QPS/VM is 11,000 QPS/node, not 1,100 I think. (2,200 x 15 VMs / 3 node).

With this change (1400 QPS/VM) 14,000 QPS/node is correct (given that there are 30 VMs and 3 nodes)

File: data/throughput_benchmarks/workloadc.yaml (hunk 1 above)
Comment (suraj-pasuparthy) [reply data/throughput_benchmarks/workloadc.yaml:35]: Agreed, but the intent here was to set these numbers exactly to what was used in our benchmarks. wanted to make sure with a high level of confidence that the numbers are reproducible.
going forward, once we have more numbers on Multi region and other workloads, we will be amending the workloads to reflect whatever configuration was used to get to those numbers

File: data/throughput_benchmarks/workloadc.yaml (hunk 2 above)
Comment (suraj-pasuparthy) [reply data/throughput_benchmarks/workloadc.yaml:24]: Ack

File: README.md (hunk 3 above)
Comment (suraj-pasuparthy) [reply README.md]: ok

File: data/throughput_benchmarks/workloadc.yaml (hunk 4 above)
Comment (suraj-pasuparthy) [reply data/throughput_benchmarks/workloadc.yaml:47]: yup. saw the typo :) and changed it to reflect the correct QPS / node

**Review by bfarayev — APPROVED** [review]
LGTM

**Comment by suraj-pasuparthy** [comment]
@bfarayev thanks for the review. i dont seem to have merge rights in this branch. could you help merge this please?

**Comment by suraj-pasuparthy** [comment]
@AdamBSteele could you help merge this please?
